### ISSUE TITLE
White screen issue

### ISSUE THREAD

**Issue by stepanselyuk** [harbur/kubernetic#284]
Hello,
sometimes (I would say often) when I click for example on the terminal tab, I got a white screen (whole app). Then I need to close the application and lose context where I worked (e.g. namespace, some filter).

**Comment by dkapanidis** [comment]
I will try to reproduce this, is it possible your pod is removed while you're focused on the terminal tab and this is what triggers the white screen?

**Comment by stepanselyuk** [comment]
No, pod not removed. When I restart the application, I can open the pod in the terminal.
I think maybe the application facing some exception issues and fail with a white screen.

**Comment by stepanselyuk** [comment]
E.g. just tried to remove a few ingresses, and got the white screen.

**Comment by dkapanidis** [comment]
so, you were on the Ingresses list screen, selected various ingresses to batch-remove them, got a popup for confirmation, accepted and then you got the white screen. is that correct?

**Comment by stepanselyuk** [comment]
Yes, I selected a few certificates, then the app showed me the confirmation screen. Then after some seconds, I got the white screen. Yes I'm talking about cert-manager certificates, and I know it will re-create those deleted certificates almost instantly (because of specified annotations in ingresses). Maybe this knowledge will help you.

**Comment by dkapanidis** [comment]
ok, got it, and yes cert-manager will regenerate those certificates instantly for you. I'll try to reproduce it my side, if I can't I may ask you for more feedback then.

**Comment by dominch** [comment]
I just hit same issue with cert manager (it happens always when deleting viewed cert) 
Also occurs sometimes when working on specs on some pods and services. 

blank window and nothing else than restart can be done, i tried some keyboard combinations (no luck), there is nothing interested on menu (maybe there should be something like "home" item).

**Comment by pinkfloydx33** [comment]
I get this issue fairly regularly. It can pretty much happen on any screen while performing any sort of action. The two most common occurrences for me both relate to filtering: 
- Filtering a built-in resource (ie. Deployments) when there are no results. 
- Filtering a Custom Resource (ie. Cert Manager Certificates) whether there are results or not

These will _sometimes_ result in full application white-screen that can only be resolved by closing the application. Note that these aren't the only actions that cause it to happen, but they are the ones I *do* notice as the biggest offenders.

**Comment by rgwave** [comment]
Same issue.  Steps to reproduce using AWS CLI for kubectl on EKS.  
1. Click on pods in left frame. 
2. Click on a particular pod in the right frame.  
3. Quickly while pod info is still populating click on "specification" submenu.  Instant white screen.  No way to go back to previous screen.  And nothing of note in the kubernetic application logs.

Easy to reproduce.  Though it make take a remote kubenetes server to reproduce the network latency.
